providers/evercrypt/fst/EverCrypt.Hash.Incremental.fst

```diff
@@ -75,6 +75,7 @@ let evercrypt_hash : block hash_alg =
     Hacl.Streaming.MD.max_input_length64
     hash_len
     block_len
+    block_len // No vectorization
```

Comment:
it would be interesting to have a larger buffer here and allow vectorization on the underlying blake2 implementations but that's another story